doc/ko/README.md

```diff
@@ -92,7 +92,7 @@ Note that, there's an experimental support for dynamic tracing on x86_64
 which doesn't require such (re-)compilations.  Also recent compilers have
 some options to help uftrace to reduce tracing overhead with similar way
 (although it still needs recompilation of your program).  Please see
-[doc/uftrace-record.md](doc/uftrace-record.md) file.
+[uftrace-record.md](uftrace-record.md) file.
```

Comment:
I think it needs to keep original path in the square bracket.

Reply:
I just fixed it. Thanks.